### ISSUE TITLE
Most values are wrong

### ISSUE THREAD

**Issue by IAmOrion** [skpang/teensy_ecu_simulator#1]
I tested this the other day when I needed to use an OBD Emulator to test some code.

To check the code for this was working, I first tested with OBD Auto Doctor, Dash Command and Torque Pro.

The values are all wrong though.

RPM on Teensy 3660 -- RPM in  software 760
Coolant Temp on Teensy 0 -- In software -40
Coolant Temp on Teensy 114c -- In software 70.6C
Throttle % on Teensy 44% -- in software 18%

The ONLY value that actually works as expected is vehicle speed!.

114 km/h correctly showed as 70-71mph

So basically, all values shown on the TFT are WRONG, all of the OBD Software (Torque Pro, OBD Auto Doctor, Dash Command) read much lower values.  (The values are consistent across the apps, so it's not an app problem)

**Comment by skpang** [comment]
Fixed TFT real values, apart from O2 voltage.
